File: Clases2023-24/01-Repaso-Unidad1-5/app/src/main/java/com/example/repasoi/Usuario.java
package com.example.repasoi;

public class Usuario {
    private String nombre;

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public Usuario(String nombre) {
        this.nombre = nombre;
    }
    public Usuario() {

    }
    @Override
    public String toString() {
        return "Usuario{" +
                "nombre='" + nombre + '\'' +
                '}';
    }
}

File: Clases2023-24/05-RepasoAplicacionGastos/app/src/main/java/com/example/repasoaplicaciongastos/entities/Usuario.kt
package com.example.repasoaplicaciongastos.entities

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "usuario")
data class Usuario(
    @PrimaryKey(autoGenerate = true) var idUsuario: Int,
    @ColumnInfo var nombre :String,
    @ColumnInfo var contrasena:String) {
    constructor(nombre :String, contrasena:String) : this(0,nombre,contrasena)

    override fun toString(): String {
        return "Usuario(nombre='$nombre', contrasena='$contrasena', idUsuario=$idUsuario)"
    }
}
